### PR TITLE
US026 - Add Branding to the Product

### PR DESCRIPTION
**Pull request by CharlesM988**
### Added the Kainos branding to the Web App

**This is not adding any functionality - only the branding to the UI.**

**Branding additions include:**

- Colours
- Fonts
- Logo
- Images
- Favicon

**Additions and Changes:**

- Updated index.html to include the new branding.
- Updated package.json to include Bootstrap.
- Created template-placeholder.html that includes templates for the rest of the team to use (this is not public facing).
- Created styles.css to override the default Bootstrap themes - this was placed within public/css/.
- Create a scripts.js for future use - this was placed within public/.
- Added an image, a logo and a favicon to use - these were places within public/images/.

Screenshot 2023-12-06 at 11 33 10

Screenshot 2023-12-06 at 11 33 30

### CODE REVIEW

**Review by Cormac-F — APPROVED** [review]
LGTM

File: public/script.js
Comment (jkelso662) [inline public/script.js:1]: Is this needed?

File: .gitignore [hunk 1]
```diff
@@ -1288,3 +1288,509 @@ node_modules/yallist/README.md
...
+node_modules/@popperjs/core/dist/esm/types.js
+node_modules/@popperjs/core/dist/esm/dom-utils/contains.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getBoundingClientRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getClippingRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getCompositeRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getComputedStyle.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getDocumentElement.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getDocumentRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getHTMLElementScroll.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getLayoutRect.js
```
Comment (jkelso662) [inline .gitignore]: 😮  wow! These are all caught by just `node_modules/` I think!

**Review by jkelso662 — DISMISSED** [review]
Couple of wee things but otherwise, UI looking slick! Really like it 😄

File: public/script.js
Comment (CharlesM988) [reply public/script.js:1]: @jkelso662 No this technically isn't I'll remove it for now.

File: .gitignore (hunk 1 above)
Comment (CharlesM988) [reply .gitignore]: @jkelso662 Seems to be, I'll update this to exclude everything within node_modules/

**Review by lullabelle — APPROVED** [review]

**Pull request merged**